etc/disable-programs.inc

```diff
@@ -44,7 +44,27 @@ blacklist ${HOME}/.openshot_qt
 blacklist ${HOME}/.flowblade
 blacklist ${HOME}/.config/flowblade
 blacklist ${HOME}/.config/eog
-
+blacklist ${HOME}/.config/arkrc
+blacklist ${HOME}/.config/atril
```

Comment:
Aren't a lot of these duplicates? For example, 
`blacklist ${HOME}/.config/atril` is on lines 20 and 48
`blacklist ${HOME}/.config/aweather` is on 18 and 49
`blacklist ${HOME}/.config/Cryptocat` is on 53 and 137
`blacklist ${HOME}/.local/share/gnome-chess` is on 146 and 194

There may be some others but that's all I noticed at first glance.